sandbox/main.lua

```diff
@@ -27504,9 +27508,9 @@ function tsarina(enemyFaction)
 		setWeaponStorage("EMP", 0):
 		setWeaponStorage("Homing", 0):
 		setWeaponStorage("HVLI", 100):
-		setHull(600):
-		setShields(100, 100, 100):
-		setAI("fighter"):
```

Comment:
Setting an AI for a player ship does not make sense since the players will be controlling it, not the AI stack

Reply:
I think this was a display bug of github?...

Reply:
It was more me seeing a player ship change above and this change looking like it was in a player ships change